pkgs/development/python-modules/mygene/default.nix

```
@@ -0,0 +1,37 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  setuptools,
+  biothings-client,
+  requests,
+  with-pandas ? false,
+  pandas,
+}:
+buildPythonPackage rec {
+  pname = "mygene";
+  version = "3.2.2";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "biothings";
+    repo = "mygene.py";
+    rev = "v${version}";
+    hash = "sha256-/KxlzOTbZTN5BA0PrJyivVFh4cLtW90/EFwczda61Tg=";
+  };
+
+  build-system = [ setuptools ];
+  dependencies = [
+    biothings-client
+    requests
+  ] ++ (if with-pandas then [ pandas ] else [ ]);
```

```suggestion
  ];
```
Please use `optional-dependencies` if necessarry.